CI/wininstaller/build_installer.cmd

```diff
@@ -0,0 +1,95 @@
+@echo off
+title VCMI Installer Builder
+setlocal enabledelayedexpansion
+cls
+
+REM Define variables dynamically relative to the normalized base directory
+set "AppVersion=1.6.1"

```

Comment:
Can this be updated automatically in some way?

Reply:
Updated script to parse these 4 needed variables as params.

`Build_Installer.cmd AppVersion AppBuild InstallerArch VCMIFolder`